stores/mssql/docker-compose.perf.yaml

```diff
@@ -0,0 +1,16 @@
+services:
+  db:
+    image: mcr.microsoft.com/mssql/server:2022-latest
+    container_name: 'mssql-perf-test-db'
+    ports:
+      - "1435:1433"
+    environment:
+      - ACCEPT_EULA=Y
+      - SA_PASSWORD=${SA_PASSWORD:-Your_password123}
+    healthcheck:
+      test: ["CMD-SHELL", "echo > /dev/tcp/localhost/1433"]
+      interval: 10s
+      timeout: 5s
+      retries: 10
+      start_period: 20s
+    # shm_size: 1gb  # Optional: uncomment if you want to set shared memory size 
```

Comment:
We probably don't need this file. For our codebase we really only have a perf file for pg, where we have perfomance tests.